src/DurableTask.Netherite/DurableTask.Netherite.csproj

```diff
@@ -49,17 +49,17 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.38.0" />
-    <PackageReference Include="Azure.Data.Tables" Version="12.8.0" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.11.2" />
-    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.11.2" />
+    <PackageReference Include="Azure.Core" Version="1.43.0" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.9.1" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.11.5" />
+    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.11.5" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.2.3" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
-    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.16.2" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.22.1" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.17.1" />
     <PackageReference Include="Microsoft.FASTER.Core" Version="2.6.5" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-    <PackageReference Include="System.Threading.Channels" Version="4.7.1" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
+    <PackageReference Include="System.Threading.Channels" Version="6.0.0" />
```

Comment:
Do we **need** to upgrade these 2? For maximum bundles compatibility (we'll be releasing this in an existing bundles feed) I'd rather not majorly increase dependencies that we don't strictly need. We'll get another chance, with more time, to see if it's safe to increase this once we consume DF WebJobs V3, so I'd prefer to delay these major version increases until then

Reply:
ok, I will back those out

Reply:
I am getting downgrade warnings if I don't include "System.Threading.Channels" Version="6.0.0" , must be being picked up somewhere

Reply:
I think that:

- I can entirely remove reference to System.Threading.Channels because it is implicitly imported now by EH
- Sourcelink is a build-time only dependency so I don't think there is an issue with keeping it there.